test/testdata/resolver/missing_generic_type.rb

```diff
@@ -0,0 +1,13 @@
+# typed: __STDLIB_INTERNAL
+# disable-fast-path: true
```

Comment:
It is not clear to me what this does, or why this is required, but the LSP version of this test will fail if this line is not included.  Guidance appreciated.